cmd/test_gen/main.cpp

```diff
@@ -70,20 +74,20 @@ generate_crypto()
 
   // Construct a test case for each suite
   for (auto suite : suites) {
-    // HKDF-Extract
-    auto hkdf_extract_out =
-      hkdf_extract(suite, tv.hkdf_extract_salt, tv.hkdf_extract_ikm);
+    // kdf-Extract
+    auto kdf_extract_out =
+      suite.hpke->kdf->extract(tv.kdf_extract_salt, tv.kdf_extract_ikm);
```

Comment:
I think it's OK to expose that MLS is using the KDF from HPKE.  The latest push refactors to use references to static objects instead of pointers, though.